### PR TITLE
[Maintenance] `sylius/sylius-rector` instead of `rector/rector`

### PR DESCRIPTION
**Pull request by Rafikooo**

### CODE REVIEW

**Review by Zales0123 — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -52,9 +52,9 @@
         "phpstan/phpstan-webmozart-assert": "^1.1",
         "phpunit/phpunit": "^8.5",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "rector/rector": "^0.14.6",
         "stripe/stripe-php": "^6.43",
         "sylius-labs/coding-standard": "^4.0",
+        "sylius/sylius-rector": "0.11.x-dev",
```
Comment (GSadee) [inline composer.json:57]: Why do we have `0.11.x-dev` version there?

**Comment by GSadee** [comment]
Thank you, Rafał! :1st_place_medal:

File: composer.json (hunk 1 above)
Comment (Rafikooo) [reply composer.json:57]: I think both `0.11.x-dev` and `dev-main` are correct as `0.11-dev` is set as an alias

Comment (GSadee) [reply composer.json:57]: My point was that SyliusRector has no released version yet, so why is it targeting 0.11 instead of 0.1 (or possibly 1.0)?

Comment (Rafikooo) [reply composer.json:57]: I relied on [this](https://github.com/Sylius/SyliusRector/blob/main/composer.json#L48). If it's wrong I'll change it

**Pull request merged**